vaadin-messages-flow-parent/vaadin-messages-flow-integration-tests/src/test/java/com/vaadin/flow/component/messages/tests/MessageListIT.java

```diff
@@ -44,7 +44,7 @@ public void setInitialItems_messagesRendered() {
 
         MessageElement msg = messages.get(0);
         Assert.assertEquals("Unexpected text content", "foo", msg.getText());
-        Assert.assertEquals("Unexpected time prop", "2021-01-01T00:00:00Z",
+        Assert.assertEquals("Unexpected time prop", "1 Jan 2021, 01:00",
```

Comment:
These depend now on the server's locale and the client's timezone, so we need to figure out how to test them consistently in different environments. I don't have a solution in mind right away.

Reply:
Right, I'm setting the locale directly on the UI now to be consistent.

Reply:
Right! There is a new method to set a specific timezone in Selenium 4, but we're on 3. I can set times to 12:00 GMT then check the strings starts with the correct date string: should be safe for all timezones. WDYT?

Reply:
Sounds like a plan! :+1: If we want to be really safe, we should check also that the time-part is included e.g. with a regular expression, but I don't think it's a must-have.

Reply:
Should we also use `UI::setLocale` in `MessageListPage` to make sure that the locale is what we expect? I'm not sure what actually defines the default locale.

Reply:
Yeah, I'm using `UI::setLocale` to set a consistent locale and then also to change it and test time formats accordingly.

Reply:
Looks good! :+1: